dependencies/maven/artifacts.snapshot

```diff
@@ -18,12 +18,18 @@
 @maven//:com_google_errorprone_error_prone_annotations_2_3_4
 @maven//:com_google_guava_guava
 @maven//:com_google_guava_guava_23_0
+@maven//:com_google_http_client_google_http_client
+@maven//:com_google_http_client_google_http_client_1_34_2
 @maven//:com_google_j2objc_j2objc_annotations
-@maven//:com_google_j2objc_j2objc_annotations_1_1
+@maven//:com_google_j2objc_j2objc_annotations_1_3
 @maven//:com_google_protobuf_protobuf_java
 @maven//:com_google_protobuf_protobuf_java_3_5_1
+@maven//:commons_codec_commons_codec
+@maven//:commons_codec_commons_codec_1_11
 @maven//:commons_io_commons_io
 @maven//:commons_io_commons_io_2_3
+@maven//:commons_logging_commons_logging
+@maven//:commons_logging_commons_logging_1_2
```

Comment:
Let's check why we are bringing in 6 additional dependencies.

Reply:
Agreed. This doesn't look right at all- as far as I can tell, this PR doesn't add any code that uses any of them.

Reply:
They were brought in by the new implementation of `release_validate_deps`. Therefore, the dependency will not be included in the JAR. It's only used in build. See [this discussion](https://discord.com/channels/665254494820368395/672728623706275845/793452248284790814) for reference.

The fact that it's not clear which dependencies are only used for build / test and which one are actually included in the distribution will be addressed soon: https://github.com/graknlabs/dependencies/issues/244.